StoreForecasts/StoreForecasts.csproj

```diff
@@ -4,6 +4,12 @@
     <ProjectReference Include="..\Schema\Schema.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="dotenv.net" Version="3.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2-beta1" />
```

Comment:
Actually, probably don't want these in here yet.  They seem like they should be added in the webinar itself since they're used specifically for talking with the API
```suggestion
```